velox/vector/tests/VectorMaker.h

```diff
@@ -193,6 +193,22 @@ class VectorMaker {
     return flatVector;
   }
 
+  /// Create a DECIMAL FlatVector from native DECIMAL values, precision, scale.
+  ///
+  /// Elements are nullable.
+  ///
+  /// Examples:
+  ///  auto flatVector = decimalFlatVectorNullable({1, std::nullopt, 3}, 8, 1);
```

Comment:
I find 8, 1 not very readable. Would it make sense to pass DECIMAL(8, 1) instead?